aspnetcore/data/entity-framework-6.md

```diff
@@ -4,14 +4,14 @@ author: rick-anderson
 description: Entity Framework 6.3 and later works with ASP.NET Core 3.1 and later.
 ms.author: riande
 ms.custom: mvc
-ms.date: 7/24/2020
+ms.date: 7/14/2020
```

Comment:
Somehow, this date was in the future.

Reply:
I'm not surprised ... Rick has a *Tardis*. :smile: